assembly/src/ast/parsers/io_ops.rs

```diff
@@ -324,7 +324,7 @@ fn parse_long_hex_param(op: &Token, hex_str: &str) -> Result<Node, ParsingError>
 /// - The length of the chunk of a long hex string (little-endian) is not equal to 16.
 /// - If the string does not contain a valid hexadecimal value.
 /// - If the parsed value is greater than or equal to the field modulus.
-fn parse_hex_value(
+pub fn parse_hex_value(
```

Comment:
Nit: Might make sense to move these to a more general location, won't be easy to remember that this helper is in the io_ops module rather than at the top level parser module or something like that

Reply:
I was thinking about it, I thought that the primary purpose if this function is to handle hex values of the `push` instruction, so it is not worth it to move this whole function to the more general location just to handle hexes in the constants. But for me it's not clear what is the best option, so I can move it, no problem